lint-staged.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
I got tired of these rules getting formatted/unformatted constantly by eslint and prettier, so I moved them in here.